CMakeLists.txt

```diff
@@ -175,6 +175,7 @@ add_library(smackTranslator STATIC
   include/smack/MemorySafetyChecker.h
   include/smack/SignedIntegerOverflowChecker.h
   include/smack/SplitAggregateLoadStore.h
+  include/smack/GenModifies.h
```

Comment:
I think that naming this file `ModAnalysis` would be better. So update its name.